examples/exact_time_inference.jl

```diff
@@ -15,7 +15,7 @@ using TemporalGPs
 using TemporalGPs: RegularSpacing
 
 # Build a GP as per usual, and wrap it inside a TemporalGPs.jl object.
-f_raw = GP(Matern52Kernel());
+f_raw = GP(AbstractGPs.ConstMean(5.0), Matern52Kernel());
```

Comment:
```suggestion
f_raw = GP(5.0, Matern52Kernel());
```
Maybe this is cleaner / more what we'd rather see people do in the wild?